src/boot/boot.c

```diff
@@ -1787,7 +1794,9 @@ static bool config_sysfail_occured(Config *config) {
 
         assert(config);
 
-        sysfail_type = sysfail_check();
+        SysFailConfig sysfail_config = { .check_firmware_update = config->sysfail_firmware_upd};
```

Comment:
Please wrap the line like the other places:
```c
SysFailConfig sysfail_config = { 
        .check_firmware_update = config->sysfail_firmware_upd,
};
```

Reply:
fixed